nautobot/virtualization/tests/test_api.py

```diff
@@ -326,3 +326,57 @@ def setUpTestData(cls):
                 "untagged_vlan": vlans[2].pk,
             },
         ]
+
+
+class VMInterfaceTestVersion13(APIViewTestCases.APIViewTestCase):
+    api_version = "1.3"
+    model = VMInterface
+    brief_fields = ["display", "id", "name", "url", "virtual_machine"]
+    bulk_update_data = {
+        "description": "New description",
+    }
+    choices_fields = ["mode"]
+
+    @classmethod
+    def setUpTestData(cls):
+
+        clustertype = ClusterType.objects.create(name="Test Cluster Type 1", slug="test-cluster-type-1")
+        cluster = Cluster.objects.create(name="Test Cluster 1", type=clustertype)
+        virtualmachine = VirtualMachine.objects.create(cluster=cluster, name="Test VM 1")
+
+        VMInterface.objects.create(virtual_machine=virtualmachine, name="Interface 1")
+        VMInterface.objects.create(virtual_machine=virtualmachine, name="Interface 2")
+        VMInterface.objects.create(virtual_machine=virtualmachine, name="Interface 3")
```

Comment:
Would be good to set `status` on these objects.

Reply:
`status` is set to `active` if none was provided.

Reply:
Understood, but it would be good to set explicitly in the "new" test cases since the default behavior is really only provided for backwards-compatibility.